src/material-examples/dialog-content/dialog-content-example.ts

```diff
@@ -0,0 +1,26 @@
+import {Component} from '@angular/core';
+import {MdDialog} from '@angular/material';
+
+/**
+ * @title Dialog Content
+ */
+@Component({
```

Comment:
I addressed the scrolling and title, can you expand on what you had in mind for the button alignment?